_assets/scripts/run_unit_tests.sh

```diff
@@ -1,25 +1,45 @@
 #!/usr/bin/env bash
 set -o pipefail
+source "${GIT_ROOT}/_assets/scripts/colors.sh"
+
+if [[ $UNIT_TEST_FAILFAST == 'true' ]]; then
+  GOTEST_EXTRAFLAGS="${GOTEST_EXTRAFLAGS} --failfast"
+fi
+
+if [[ -z "${UNIT_TEST_COUNT}" ]]; then
+  UNIT_TEST_COUNT=1
+fi
 
 GIT_ROOT=$(cd "${BASH_SOURCE%/*}" && git rev-parse --show-toplevel)
 
-source "${GIT_ROOT}/_assets/scripts/colors.sh"
+last_failing_exit_code=0
 
 for package in ${UNIT_TEST_PACKAGES}; do
   echo -e "${GRN}Testing:${RST} ${package}"
   package_dir=$(go list -f "{{.Dir}}" "${package}")
   output_file=${package_dir}/test.log
 
-  go test -tags "${BUILD_TAGS}" -timeout 30m -v -failfast "${package}" ${GOTEST_EXTRAFLAGS} | \
-     if [ "${CI}" = "true" ]; then cat > "${output_file}"; else tee "${output_file}"; fi
+  go test -count="${UNIT_TEST_COUNT}" -tags "${BUILD_TAGS}" -v "${package}" ${GOTEST_EXTRAFLAGS} | \
+     if [[ "${CI}" == 'true' ]]; then cat > "${output_file}"; else tee "${output_file}"; fi
```

Comment:
This line looks like it could be a function.

Reply:
Done!